### PR TITLE
Statement on ECU encryption--closes Issue #7

### PR DESCRIPTION
**Pull request by jhdalek55**
I added the above, based on Ira's recommendation. @iramcdonald, please make sure my wording is correct. I'm not sure I captured this correctly. Reviews by others are appreciated. as well.

### CODE REVIEW

File: ecus.md [hunk 1]
```diff
@@ -45,3 +45,7 @@ If you choose to use symmetric ECU keys, it would be a good idea to store the ke
 2. Given an ECU identifier and an image identifier, encrypt the image for that ECU.
 
 Unencrypted images should be loaded onto the symmetric key server by some out-of-band physical channel (for example, via USB stick).
+
+## Encryption of images on ECUs
+
+The Director repository may encrypt images if required ([Section 5.3.2 of the Uptane Standard](https://github.com/uptane/uptane-standard/blob/master/uptane-standard.md#director-repository-director_repository). However, no Uptane implementation should support interactive requests from an ECU for encryption.
```
Comment (trishankatdatadog) [inline ecus.md]: That's an interesting sentence at the end. Could someone explain why it was added?

Comment (jhdalek55) [reply ecus.md]: On our last phone meeting, Ira recommended this statement should be made. All of this is  in response to issue #7 . if there is a problem, please share.

Comment (trishankatdatadog) [reply ecus.md]: No problem, Lois, just curious. I'm guessing there are some legal requirements or something similar I'm not aware of.

**Comment by iramcdonald** [comment]
Hi Trishank,

(Saving you from reading issue #7)

The sentence was added because adding a command to allow the
Target ECU to explicitly request an encrypted image at Download
time, means adding a command that requires strong authentication,
because it could also be used to turn *off* encryption (allowing the
easy reverse engineering of unencrypted firmware and theft of IPR).
This new control plane command also increases the attack surface.

Only the OEM and their suppliers should determine the policy on
encrypting particular binaries.  It should be configured for use by
Director, not toggled by the Target ECU.

Cheers,
- Ira

Ira McDonald (Musician / Software Architect)
Co-Chair - TCG Trusted Mobility Solutions WG
Co-Chair - TCG Metadata Access Protocol SG
Chair - Linux Foundation Open Printing WG
Secretary - IEEE-ISTO Printer Working Group
Co-Chair - IEEE-ISTO PWG Internet Printing Protocol WG
IETF Designated Expert - IPP & Printer MIB
Blue Roof Music / High North Inc
http://sites.google.com/site/blueroofmusic
http://sites.google.com/site/highnorthinc
mailto: blueroofmusic@gmail.com
PO Box 221  Grand Marais, MI 49839  906-494-2434

**Comment by trishankatdatadog** [comment]
Thanks, Ira. (Wasn't aware this was explained in #7, my bad.) This makes sense, and should be added to the text.

**Comment by jhdalek55** [comment]
@iramcdonald  Maybe, for the sake of clarity, i should add your clarification  after that last sentence? I've just reshaped your explanation a bit.

A command that allowed the
Target ECU to explicitly request an encrypted image at download
time not only increases the attack surface, but could also be used to turn *off* encryption. This which would make it easy to  reverse engineer unencrypted firmware and steal IPR. Only the OEM and its suppliers should determine policy on
encrypting particular binaries, and this policy should be configured for use by the 
Director repository rather than being toggled by the Target ECU.

**Comment by iramcdonald** [comment]
Hi Lois,

Much better.  Good thing somebody around here can write
lively, clear English...

Cheers,
- Ira

Ira McDonald (Musician / Software Architect)
Co-Chair - TCG Trusted Mobility Solutions WG
Co-Chair - TCG Metadata Access Protocol SG
Chair - Linux Foundation Open Printing WG
Secretary - IEEE-ISTO Printer Working Group
Co-Chair - IEEE-ISTO PWG Internet Printing Protocol WG
IETF Designated Expert - IPP & Printer MIB
Blue Roof Music / High North Inc
http://sites.google.com/site/blueroofmusic
http://sites.google.com/site/highnorthinc
mailto: blueroofmusic@gmail.com
PO Box 221  Grand Marais, MI 49839  906-494-2434

**Comment by jhdalek55** [comment]
I'll spell out the IPR and I'll see if I can fix the line breaks.

**Comment by jhdalek55** [comment]
Addressed comments from @patrickvacek. Could we get one more approval so we can merge? Thanks.

Lois

**Comment by patrickvacek** [comment]
Looks good, thanks!

**Pull request merged**